USAGE.rst

```diff
@@ -0,0 +1,137 @@
+kafkacat
+========
```

Comment:
Can you rename the file as `kafkacat-usage.rst`? This will make more sense when it's imported to CP docs as standalone file.